src/cli/cmd/harvest/platforms/platforms.go

```diff
@@ -214,16 +214,18 @@ func isPlatformYAML(filename string) bool {
 func findPlatformYAML(directoryPath string) (string, error) {
 	var foundPath string
 
-	extensions := map[string]bool{".yaml": true, ".yml": true}
+	filenames := []string{"platform.yaml", "platform.yml"}
```

Comment:
In this case I would probably make a. "top-level 'constant' variable" allowedPlatformFileNames or something.

Reply:
Thanks for the suggestion. Have made the change